discovery/transport/http/src/DiscoveryTransportHandlerHttp.cs

```diff
@@ -81,11 +82,11 @@ public override async Task<string> IssueChallengeAsync(DiscoveryTransportIssueCh
 
                 var onboardRequest = new ChallengeRequest(
                     registrationId, 
-                    Convert.ToBase64String(securityProvider.GetEndorsementKey()), 
-                    Convert.ToBase64String(securityProvider.GetStorageRootKey()));
+                    securityProvider.GetEndorsementKey(), 
+                    securityProvider.GetStorageRootKey());
 
                 Challenge challenge = await client.DiscoveryRegistrations
-                    .IssueChallengeAsync(onboardRequest, cancellationToken)
+                    .IssueChallengeAsync("2023-12-01-preview", onboardRequest, cancellationToken: cancellationToken)
```

Comment:
define a const with the apiVersion = "2023-12-01-preview". Assuming I read that right...